iceoryx_posh/test/moduletests/test_roudi_shm.cpp

```diff
@@ -320,11 +336,19 @@ TEST_F(PortManager_test, PortDestroy)
     iox::capro::ServiceDescription cap2(2, 2, 2);
 
     // two processes p1 and p2 each with a sender and receiver that match to the other process
-    auto senderData1 = m_shmManager->acquireSenderPortData(cap1, p1, m_payloadMemoryManager).get_value();
-    auto receiverData1 = m_shmManager->acquireReceiverPortData(cap2, p1);
-
-    auto senderData2 = m_shmManager->acquireSenderPortData(cap2, p2, m_payloadMemoryManager).get_value();
-    auto receiverData2 = m_shmManager->acquireReceiverPortData(cap1, p2);
+    auto senderData1 = m_shmManager
```

Comment:
We could construct p1 and p2 to be CString100 to avoid some code I think.

Reply:
Yeah, you're right. Here we don't need the `cxx::string::operator+()`